src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -635,6 +635,36 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
             }
         }
 
+        [Fact]
+        public async Task GetAsync_AllowAutoRedirectTrue_RedirectWithoutLocation_ReturnsOriginalRequest()
```

Comment:
"ReturnsOriginalRequest" should be named "ReturnsOriginalResponse", i.e. it doesn't follow the redirect since it doesn't know where to go.

Reply:
Changed.